smokefree/sites/all/themes/smokefreegov/src/scss/footer.scss

```diff
@@ -339,6 +339,12 @@ section#block-boxes-live-char-bar {
 	}
 }
 
```

Comment:
This looks good. Do you mind adding appropriate comments to the code.

Thanks!